pycbc/inference/option_utils.py

```diff
@@ -445,7 +445,7 @@ def results_from_cli(opts, load_samples=True, walkers=None):
     if load_samples:
         logging.info("Loading samples")
         # check if need extra parameters for a non-sampling parameter
-        file_parameters, cs = transforms.get_conversions(
+        file_parameters, cs = transforms.get_standard_transforms(
```

Comment:
I don't want to call it `get_transforms` because it will not necessarily get all transforms; it can only get those they do not take input arguments. At the moment that's all transforms defined, but it won't be once I add the Logit transform. Let me update #1681 to this so you can see what I mean.

Also, this function is only used for CBC-specific transforms. That said, `standard_transforms` maybe isn't the best name. I've changed `standard_transforms` to `common_cbc_forward_transforms`, `standard_inverse_transforms` to `common_cbc_inverse_transforms`, `all_standard_transforms` to `common_cbc_transforms`, and have renamed `get_standrad_transforms` to `get_common_cbc_transforms`. That way it's clear that the function is getting both forward and inverse transforms.